netlify.toml

```diff
@@ -0,0 +1,17 @@
+[build]
+base = "site/"
+publish = "site/public/"
+command = "hugo --gc"
+
+[build.environment]
+HUGO_VERSION = "0.65.1"
+
+[context.production.environment]
+HUGO_ENV = "production"
+HUGO_ENABLEGITINFO = "true"
```

Comment:
I suspect that we don't need both the env var and the flag? Let's choose one.

Reply:
u r right.